.bento/config.yml

```diff
@@ -29,20 +29,7 @@ tools:
     - try-except-continue
     - urllib-urlopen
     run: true
-  dlint:
-    ignore: []
-    run: true
   flake8:
     ignore:
     - bare-except-bugbear
     run: true
-  r2c.registry.latest:
-    ignore: []
-    run: true
-  r2c.requests:
-    ignore:
-    - use-timeout
-    run: true
-  semgrep:
```

Comment:
Covered by `semgrep-run-r2c-config`.

Reply:
This does let semgrep run automatically as a pre-commit hook. Should we add semgrep to .pre-commit.yaml first?

Reply:
Required a bit more work, but I updated the pre-commit configuration as well.